src/shady-css/common.js

```diff
@@ -18,7 +18,8 @@ const matcher = {
   whitespace: /\s/,
   whitespaceGreedy: /(\s+)/g,
   commentGreedy: /(\*\/)/g,
-  boundary: /[\(\)\{\}'"@;:\s]/,
+  boundary: /[\(\)\{\}'"@;:\s+\-\\/*,]/,
+  word: /[^\(\)\{\}'"@;:\s+\-\\/*,]/,
```

Comment:
It seems feasible to combine these, since one is the negation of the other. This should save some small number of bytes in the final payload.